docs/public/google67c4a117546ddf5e.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google67c4a117546ddf5e.html
```

Comment:
Could you please add a new line underneath? Just to get rid of this icon in Github 
<img width="303" alt="image" src="https://user-images.githubusercontent.com/12689383/204923838-696de549-1a5e-4903-b91c-1d209d269168.png">